src/DomUpdates.js

```diff
@@ -1,29 +1,137 @@
+//imports
+import TravelAgency from './TravelAgency'
+import Traveler from './Traveler'
+// import travelers from './index'
+// import destinations from './index'
+// import trips from './index'
 
-class DomUpdates {
-    constructor() {
-        this.counter = 0;
-    }
+//Globals
+let traveler;
+let travelAgency;
 
-    cycleImages(destinations) {
-        this.counter++
-        if(this.counter === destinations.length + 1) {
-            this.counter = 0
+//Query Selectors
+const usernameInput = document.getElementById('username')
+const passwordInput = document.getElementById('password')
+const mainArea = document.querySelector('.main')
+
+const domUpdates = {
+
+    logIn() {
+        debugger
+        const usernameArray = usernameInput.value.split('')
+        const usernameID = usernameArray.splice(8, 10).join('')
+        if(usernameInput.value === 'agency' && passwordInput.value === 'travel2020') {
+            // domUpdates.instantiateTravelAgency()
+            domUpdates.displayAgencyDashboard()
+        } else if (usernameID <= 50 && passwordInput.value === 'travel2020') {
+          domUpdates.displayTravelerDashboard()
+        } else {
+          domUpdates.displayError()
         }
-        console.log(destinations[this.counter])
-        let mainArea = document.querySelector('.main')
-        mainArea.innerHTML = `<header class='welcome-message'><h2>Welcome to Travel Tracker</h2></header>
-        <section class='cycling-images'><img src="${destinations[this.counter].image}" alt="destination-image" class='cycling-images'>
-        <footer>Your Vacation Awaits!</footer></section>`
-        setTimeout(this.cycleImages, 1000, destinations);
-    }
+        event.preventDefault()
+    },
+    
```

Comment:
I like how you put the login in the DomUpdates. I have mine in the user class so this is a different way of seeing it.